testgrid/tgrun/deploy/tgrun-pod.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: tgrun
+  labels:
+    app: testgrid
+spec:
+  containers:
+  - name: tgrun
+    image: replicated/tgrun
```

Comment:
there should be a tag on this, it's going to be very hard to reproduce and troubleshoot without a tag